apps/webmin.sh

```diff
@@ -18,6 +16,32 @@ debug_mode
 # Check if root
 root_check
 
+# Check if webmin is already installed
+print_text_in_color "$ICyan" "Checking if Webmin is already installed..."
+if is_this_installed webmin
+then
+    choice=$(whiptail --radiolist "It seems like 'Webmin' is already installed.\nChoose what you want to do.\nSelect by pressing the spacebar and ENTER" "$WT_HEIGHT" "$WT_WIDTH" 4 \
+    "Uninstall Webmin" "" ON \
+    "Reinstall Webmin" "" OFF 3>&1 1>&2 2>&3)
+    
+    case "$choice" in
+        "Uninstall Webmin")
+            print_text_in_color "$ICyan" "Uninstalling Webmin..."
+            check_command apt purge webmin -y
+            msg_box "Webmin was successfully uninstalled."
```

Comment:
add `apt purge autoremove -y` to remove dependencies as well

Reply:
Okay

Reply:
this should be done.